docs/setup/cookbook/planning.md

```
@@ -1,2 +1,158 @@
+
+Responsibilities are split between the country program and the MOSIP team. The matrix below covers technical resources, setup, rollout, management and procurement.
+
+***
+
+**Category** **Activity** **Responsibility**
+
+***
+
+Technical resources Country technical team Country & field staff
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)